jgiven-gradle-plugin/src/main/java/com/tngtech/jgiven/gradle/internal/AbstractJGivenReportImpl.java

```diff
@@ -106,5 +122,14 @@ public AbstractReportGenerator createGenerator() {
         this.thumbnailsAreShown = thumbnailsAreShown;
     }
 
+    @Override
+    public Property<Boolean> getRequired() {
+        return isRequired;
+    }
+
+    @Override
+    public DirectoryProperty getOutputLocation() {
+        return null;
```

Comment:
Why does this return `null` and not `outputLocation`?

Reply:
oops :blush: . Changed it, I am still getting an NPE

Reply:
@l-1squared Can you share a stacktrace?

Reply:
@wolfs  I hope the trace I provided you via mail suffices. If not, I am happy to share a more complete log output tomorrow. However, the stacktrace I saw so far, did not help me very much.